resources/js/processes/screen-builder/screen.vue

```diff
@@ -277,7 +277,7 @@ import formTypes from "./formTypes";
         this.validationWarnings.splice(0);
 
         if (this.type === formTypes.form && !this.containsSubmitButton()) {
-          this.validationWarnings.push(this.$t('Form requires a submit button to be saved'));
+          this.validationWarnings.push('Warning: Screens without save buttons cannot be executed.');
```

Comment:
I removed the `$t` translator function as it truncates text before and including the colon (`:`). There also didn't appear to be an entry for this in the `en.json` file.